src/lib/bootloader/grub2_widgets.rb

```diff
@@ -911,6 +918,28 @@ def init_custom_devices(custom_devices)
       end
     end
 
+    # Validates list of devices
+    #
+    # @param devs_list[String] comma separated list of device definitions
+    def valid_custom_devices?(devs_list)
+      # almost any byte sequence is potentially valid path in unix like systems
+      # AY profile can be generated for whatever system so we cannot decite if
+      # particular byte sequence is valid or not
+      return true if Mode.config
+
+      devs_list.split(",").all? do |d|
+        dev_path = DevicePath.new(d)
+
+        if Yast::Mode.installation
+          # uuids are generated later by mkfs, so not known in time of installation
+          # so whatever can be true
+          dev_path.uuid? || dev_path.label?
```

Comment:
ugh, what? This means that only uuid or label is valid in custom devices, which is definitively correct as you can specify even kernel names or by-id or by-path here.